src/vmm/src/version_map.rs

```diff
@@ -17,6 +21,13 @@ lazy_static! {
     pub static ref VERSION_MAP: VersionMap = {
         let mut version_map = VersionMap::new();
         version_map.new_version().set_type_version(DeviceStates::type_id(), 2);
+
+        #[cfg(target_arch = "aarch64")]
+        {
+            version_map.new_version().set_type_version(VcpuState::type_id(), 2);
+            version_map.new_version().set_type_version(VmState::type_id(), 2);
```

Comment:
Calling new_version() twice creates v0.25 and v0.26, this is not what we want.
Did VcpuState/Vmstate change and are now at version 2 ? To me it looks like they are version 1, and by default we can consider them existing even in FC v0.23 to keep things simple.

Reply:
To me it looks safe to drop entire aarch64 block, so we'll have an identical version map at least for now.

Reply:
You are right about calling `new_version()` twice, that is not the behavior I was aiming for. It should have been something like:
```suggestion
            version_map.set_type_version(VcpuState::type_id(), 2);
            version_map.set_type_version(VmState::type_id(), 2);
```

The [VcspuState](https://github.com/firecracker-microvm/firecracker/pull/2351/files#diff-2ce0ab493c25dcb75775ffa4c09945862fe4d5cf1ff21ce8b7acae8472bce45dR180) and [VmState](https://github.com/firecracker-microvm/firecracker/pull/2351/files#diff-d4f9b6b89be9c442cae573c191e75771391e6fe009b6a39dc12a47a4c33fa7c1R345) structures have been modified for aarch64 and this is why I thought I would be a good idea to register these changes into the version map, as well.

Reply:
However, I have nothing against your idea to consider the two structures existing in v0.23.0. I will discard that commit.